### PR TITLE
fix: Get Started button not linked to specific area of the Meshery docs

### LINKED ISSUE #504: [features page] Get Started button not linked to specific area of the Meshery docs
**Description**
The four **“Get Started”** don't point to the specific area of the Meshery docs relevant to that category.
https://meshery.io/features

**Expected Behavior**
It should link to each specific area of docs.

**Enviroment:**
Everything

---

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by afzal442**
**Description**
Get Started buttons now link to a specific area of its docs

This PR fixes #504 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by afzal442** [comment]
@warunicorn19 plz review it.😅

https://deploy-preview-508--mesheryio-preview.netlify.app/features

**Review by warunicorn19 — DISMISSED** [review]
This is great, thanks @afzal442 😄.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Let's ensure that none of the new links include anchors in them, but refer to the top-level page itself; e.g. #lifecycle-management can be removed from the lifecycle link.

File: collections/_pages/features.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
       - text:
         highlight:
     call_to_action:
-      link: /#getting-started
+      link: https://docs.meshery.io/functionality/lifecycle-management#configuration-management
```
Comment (afzal442) [inline collections/_pages/features.html:53]: to link configuration feature, I'm not sure about the proper link. plz suggest here.

File: collections/_pages/features.html [hunk 2]
```diff
@@ -86,7 +86,7 @@
         highlight: false
         featureli: true
     call_to_action:
-      link: /#getting-started
+      link: https://docs.meshery.io/functionality/filter-management#q-what-is-wasm-
```
Comment (afzal442) [inline collections/_pages/features.html:89]:
```suggestion
      link: https://docs.meshery.io/functionality/filter-management
```
Should I make the link as above?

**Comment by warunicorn19** [comment]
> Let's ensure that none of the new links include anchors in them, but refer to the top-level page itself; e.g. #lifecycle-management can be removed from the lifecycle link.

@leecalcote considering that lifecycle and configuration lead to the same page, configuration could have an anchor, right? or would it just lead to the lifecycle page itself?

File: collections/_pages/features.html (hunk 1 above)
Comment (leecalcote) [reply collections/_pages/features.html:53]: @afzal442, yikes! You're right. Let's get an issue opened in the Meshery Docs to create a new page for configuration management.

**Comment by leecalcote** [comment]
> > Let's ensure that none of the new links include anchors in them, but refer to the top-level page itself; e.g. #lifecycle-management can be removed from the lifecycle link.
> 
> @leecalcote considering that lifecycle and configuration lead to the same page, configuration could have an anchor, right? or would it just lead to the lifecycle page itself?

Yes, you're right - the anchor for now. We're in need of a dedicated page.

File: collections/_pages/features.html (hunk 2 above)
Comment (leecalcote) [inline collections/_pages/features.html]:
```suggestion
      link: https://docs.meshery.io/functionality/filter-management
```

File: collections/_pages/features.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 layout: page
 permalink: /features
 title: Features
-heading: Meshery is <i>the</i> service mesh management plane
+heading: Meshery is the service mesh management plane
```
Comment (leecalcote) [inline collections/_pages/features.html:5]:
```suggestion
heading: Meshery is <i>the</i> service mesh management plane
```

Comment (leecalcote) [reply collections/_pages/features.html:5]: Reverting this change.

File: collections/_pages/features.html (hunk 1 above)
Comment (leecalcote) [reply collections/_pages/features.html:53]: Do you mind opening this docs issue, @afzal442?

**Review by leecalcote — APPROVED** [review]

File: collections/_pages/features.html (hunk 1 above)
Comment (afzal442) [reply collections/_pages/features.html:53]: Ohk sure. Do you mean the objective will be to transfer the content of configuration-management to another page or just create a new page describing config management?

**Pull request merged**